documentation/api/guides/https.md

```diff
@@ -0,0 +1,48 @@
+# HTTPS
```

Comment:
This might be another good clarification as this only refers to local HTTPS testing configuration and has essentially nothing to do with the HTTPS configuration in our live environments.

```suggestion
# Local HTTPS testing
```